src/wrappers.rs

```diff
@@ -394,14 +394,14 @@ impl<T: Unaligned> Deref for Unalign<T> {
 
     #[inline(always)]
     fn deref(&self) -> &T {
-        Ptr::from_ref(self).transparent_wrapper_into_inner().bikeshed_recall_aligned().as_ref()
+        Ptr::from_ref(self).transmute().bikeshed_recall_aligned().as_ref()
```

Comment:
Bad news: This fails because it requires `T: KnownLayout`, which we wanted to avoid. We might be able to work around it with the `SizedKnownLayout` trick, or we might need to reconsider the `T: KnownLayout` bound.